nncf/common/tensor.py

```diff
@@ -44,4 +44,4 @@ def device(self) -> DeviceType:
         pass
 
     def is_empty(self) -> bool:
-        return False
+        return 0 in self.shape
```

Comment:
Maybe it is better to make this method abstract and implement this backend-specific function in each separate backend?

Reply:
done

Reply:
make abstract method and added implementations for ONNX and TF